### DISCUSSION TITLE
Polyhierarchy in Dendron?

### DISCUSSION THREAD

**Discussion by Matt321456** [dendronhq/dendron#3797] [Q&A]
Hello, any way to do polyhierarchy in Dendron or any workarounds? I'm thinking it's the logical extension to what Dendron already offers and values in PKM Polyhierarchy = The same node having two or more parents within a a hierarchy, allowing for complex taxonomies to intersect. 

Here's a simple example: interests.music VS field.music

".music" would be referring to the same node in the above, meaning it would inherit from each parent. Otherwise, I would simply use hyperlinks.

Thank you

**Comment by ScriptAutomate** [comment]
This may be something similar to an extension of the Alias notes feature request:
- https://github.com/dendronhq/dendron/issues/128
